### ISSUE TITLE
Please update release to 1.5.6(x) , need Waveshare_3 support

### ISSUE THREAD

**Issue by microbmen** [evilsocket/pwnagotchi#1156]
Waveshare V3 is all over Amazon, and many people are buying the V3 display.  Version 1.5.5 does not have support for V3 in it;
i.e. ui.display.type = "waveshare_3"

Please bundle a new image so that the display can be set to V3; The github repo does have support for _3, and everything works great when building fresh.  The image needs to be updated.  I think this will alleviate a lot of pain from many new comers to the utility;  

Thanks :-)

**Comment by D337z** [comment]
Surprisingly, it actually DOES have it, but the image uses such old resources that it can't work properly.  Your best bet is to compile it from source, but even then it's going to take a few sleepless nights since the master source is...very broken to put it nicely.  It's based off of 2019 work, so it will take a massive update just to get waveshare_v3 working via an image file.

**Comment by microbmen** [comment]
sadly, i know what you mean.  i built it from source. 

a much quicker method, which worked well was to take the 1.5.5 image, deploy it, then do a git clone, and `sudo pip3 install .` , which copied over the updated waveshare_3 files.    Worked perfectly.

**Comment by HasseM** [comment]
This is indeed a major problem for new users. They buy a V2.1 waveshare assuming it is a V2 but then after lot's of struggling to get it to work they find out it is a V3 and no support at all....

Do we need to crowd-fund a V3 to the author for an updated image?

**Comment by D337z** [comment]
I'm actually trying to build a fully updated version myself which includes the V3 support, updated and optimized binaries, and a working tensorflow for the ARMv6k+fp processor.  But I'm stuck at Bazel.  I'll try working with the main image and updating it as you did to see if it gives me anything more to work with, but the entire system needs updated and worked on to at least work on the intended hardware properly which is likely why it hasn't been done.

**Comment by virtadpt** [comment]
It might be time for a fork, if not a straight-up scratch build with brand-new everything.

**Comment by D337z** [comment]
That's partially what I'm working on.  I'm trying to make the 32-bit ARM compatible images, but I'm having to work from scratch on a few of the binaries since they only support ARCH64.  On the brighter side, I'm hoping to upgrade this to a 64-bit port to open up some speed for the AI training for supported systems.  Everything I'm finding warns me to only use pre-trained Tensorflow, but it defeats the entire purpose of the project unless reworked to capture, bring to multi-cored base (Pi3, Zero2, or 4), train, and upload the updated model to be used in the field once more.
On the bright side, I'm getting more optimal binaries out of it with newer versions of OpenJDK and GCC than the original afforded, but it's a slow process as I'm not great at cross-compiling for other architectures.

**Comment by badwolfunit** [comment]
Yeah i know no ones used this issue in a long time but i would just like to add how i have a v4 and that's not supported either which is super annoying. But a cool idea would be to make a mixture between a petition and a donation pot so it not only makes it even more obvious how many people want it but the donation side of it gives the devs an initiative to start the compilation of the image since it would be quite a big job and it would also let people know how much people value their work. it could be a good idea
